### PR TITLE
Chore: Replaces `Alert` FSUI

### PR DESCRIPTION
**Pull request by renatamottam**
> `@faststore/ui` package needs to be updated after merge of https://github.com/vtex/faststore/pull/1590

## What's the purpose of this pull request?
- [x] Uses `Alert` from the `faststore/ui`
- [x] Remove `Alert` folder

## How does it work?
Everything should look as before. (Visually)

## How to test it?
You can check the fixed `Alert` on any page and `Alert` inside `CartSidebar`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "@faststore/components": "2.0.30-alpha.0",
     "@faststore/graphql-utils": "^2.0.3-alpha.0",
     "@faststore/sdk": "^2.0.3-alpha.0",
-    "@faststore/ui": "2.0.37-alpha.0",
+    "@faststore/ui": "https://pkg.csb.dev/vtex/faststore/commit/4892bbe4/@faststore/ui",
```
Comment (hellofanny) [inline package.json:37]: 🎗️ Remind to update the package version when available.

File: src/Layout.tsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { lazy, Suspense } from 'react'
 import type { PropsWithChildren } from 'react'
 
-import Alert from 'src/components/common/Alert'
+import { Alert as UIAlert } from '@faststore/ui'
```
Comment (hellofanny) [inline src/Layout.tsx:4]: @renatamottam We should use the `Alert` from `common/Alert` here. 
Can you switch back?

**Comment by hellofanny** [comment]
Build is failing because https://github.com/vtex/faststore/pull/1590 hasn't Dropdown component updates yet.

File: package.json [hunk 3]
```diff
@@ -31,10 +31,10 @@
     "@envelop/parser-cache": "^2.2.0",
     "@envelop/validation-cache": "^2.2.0",
     "@faststore/api": "^2.0.3-alpha.0",
-    "@faststore/components": "^2.0.41-alpha.0",
+    "@faststore/components": "https://pkg.csb.dev/vtex/faststore/commit/ae541335/@faststore/components",
```
Comment (eduardoformiga) [inline package.json:34]: 🎗️

File: package.json [hunk 4]
```diff
@@ -31,10 +31,10 @@
...
     "@faststore/graphql-utils": "^2.0.3-alpha.0",
     "@faststore/sdk": "^2.0.3-alpha.0",
-    "@faststore/ui": "^2.0.41-alpha.0",
+    "@faststore/ui": "https://pkg.csb.dev/vtex/faststore/commit/ae541335/@faststore/ui",
```
Comment (eduardoformiga) [inline package.json:37]: 🎗️

File: src/Layout.tsx [hunk 5]
```diff
@@ -18,7 +18,11 @@ function Layout({ children }: PropsWithChildren) {
 
   return (
     <>
-      <Alert icon="Bell" link={{ text: 'Buy now', to: '/office' }} dismissible>
+      <Alert
+        icon="Bell"
+        link={{ text: 'Buy now', to: '/office', target: '_self' }}
```
Comment (emersonlaurentino) [inline src/Layout.tsx:23]:
```suggestion
        link={{ children: 'Buy now', href: '/office', target: '_self' }}
```

**Review by emersonlaurentino — APPROVED** [review]

**Review by hellofanny — APPROVED** [review]
LGTM! 🚀

**Pull request merged**